internal/lib/lib.go

```diff
@@ -63,12 +63,28 @@ func GetNamePrefix() string {
 }
 
 var DisableSync bool
+var layer7Only bool
 
 func SetDisableSync(state bool) {
 	DisableSync = state
 	utils.AviLog.Infof("Setting Disable Sync to: %v", state)
 }
 
+func SetLayer7Only(val string) {
+	if boolVal, err := strconv.ParseBool(val); err == nil {
+		layer7Only = boolVal
+
+	} else {
+		// If the variable could not be converted, default to this being false
+		layer7Only = false
```

Comment:
Nit: No need to set the false value here, as in case of error, `ParseBool` returns a false anyway.